addon/styles/_variables.scss

```diff
@@ -0,0 +1,19 @@
+//
+// Variables used by
+//
+
+
+$frost-color-nav-bar-bg: $frost-color-night-2 !default;
+$frost-color-nav-border-left: rgba($frost-color-white, .2) !default;
+$frost-color-nav-category-border-right: rgba($frost-color-white, .2) !default;
+$frost-color-nav-category-active-bg: rgba(39, 36, 37, .5) !default;
```

Comment:
We should have a TODO here to make this a colour variable if it isn't one already